### PR TITLE
Remove liquidvoting component

### PR DESCRIPTION
**Pull request by davefrey**
You will also need to remove the 'liquidvoting' row from the database, or you will get something like:

ActionView::Template::Error (undefined method `decidim_participatory_process_liquidvoting' for #<Decidim::EngineRouter:0x00007facca323d30>

The last segment 'liquidvoting' of that method name is dynamically retrieved from the database,
from decidim_components.manifest_name; changing the value changes the method name lookup.
Removing the row lets things work again.

This seems to leave the code broken in exactly the same ways as before: view rendering fails on
current_settings etc

### CODE REVIEW

**Review by oliverbarnes — CHANGES_REQUESTED** [review]
Now that the code is maturing beyond a spike, it'd be good to remove the commented code and all the TODOs - these are better captured and followed-up on through issues or issue comments.

**Review by jinjagit — APPROVED** [review]
LGTM - but I confess I haven't pulled & tested.

**Comment by davefrey** [comment]
Definitely I don't like the `TODO`s and I expect most to go away quickly, and I agree issues are better than `TODO`s.

For the moment they're my placeholders as I better understand things; it feels premature to issue-ize them. Give me a few more commits on this PR :-) 

Also it seems I've got some PRs maybe out of order; this PR will be cleaner after #42 and then #39 are merged.

**Comment by oliverbarnes** [comment]
Hehe, sure! But it sounds like the PR is draft, then?

File: app/controllers/decidim/liquidvoting/delegations_controller.rb [hunk 1]
```diff
@@ -45,14 +45,14 @@ def destroy
...
-      def current_component
-        Decidim::Component.find_by(manifest_name: "liquidvoting")
-      end
+      private
 
-      def permission_class_chain
-        [
-          Decidim::Liquidvoting::Permissions
-        ]
+      def lv_state
```
Comment (davefrey) [inline app/controllers/decidim/liquidvoting/delegations_controller.rb:50]: I'm pretty confused why this is coming back in, this was added to master some time ago; I must have made a mistake when splitting out the PRs this morning. I confirm this is the right diff now.

**Comment by davefrey** [comment]
I cleaned things up a bit, the TODOs in particular really belong to other demo work threads. I'll un-draft this.

**Review by oliverbarnes — APPROVED** [review]

**Pull request merged**